.github/workflows/BuildPR.yml

```diff
@@ -0,0 +1,27 @@
+name: PullRequest
+
+on: [pull_request]
+
+jobs:
```

Comment:
Is anything from #198 useful here? Once this works, that can be closed.

Reply:
I see no point to run with python there
`run: python3 Scripts/build.py release`

I will close it